website/content/docs/auth/gcp.mdx

```diff
@@ -224,6 +259,61 @@ account to impersonate any service account in the GCP project where it resides.
 See [Managing service account impersonation](https://cloud.google.com/iam/docs/impersonating-service-accounts)
 for more information.
 
+## Plugin Workload Identity Federation (WIF)
+
+<EnterpriseAlert product="vault" />
+
+The GCP auth engine supports the plugin WIF workflow and has a source of identity called
```

Comment:
Shouldn't we use "auth **method**" instead of "auth **engine**" for consistency?

```suggestion
The GCP auth method supports the plugin WIF workflow and has a source of identity called
```